### PR TITLE
Get-Domain parser

### PR DESCRIPTION
**Pull request by ArtificialErmine**
Added get-domain parser for os/hostnames

### CODE REVIEW

File: app/parsers/gdomain.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+from app.objects.c_relationship import Relationship
+from plugins.stockpile.app.parsers.base_parser import BaseParser
+from app.utility.logger import Logger
+
+class Parser(BaseParser):
+
+    def __init__(self, parser_info):
+        self.mappers = parser_info['mappers']
+        self.used_facts = parser_info['used_facts']
+        self.log = Logger('parsing_svc')
```
Comment (privateducky) [inline app/parsers/gdomain.py]: parsing_svc is no more

Comment (khyberspache) [reply app/parsers/gdomain.py]: I don't know how I missed that :|

Comment (khyberspache) [reply app/parsers/gdomain.py]: Bump on edit above

**Comment by privateducky** [comment]
@khyberspache bumping this PR. looks like it just needs to be re-tested, post us swapping to objects

**Comment by khyberspache** [comment]
@privateducky This PR was made after OO change-over... we were waiting on a decision on whether you cared about the logger string.  I changed it from `parsing_svc` to `Parser`, but ultimately it doesn't make as long as other parsers that want to log use the same string.

**Pull request merged**